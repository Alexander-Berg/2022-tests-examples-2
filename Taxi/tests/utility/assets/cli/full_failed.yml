options:
  base_url: https://ya.ru
  browser: chrome
  list:
    - browser: chrome
      selenium_hub_url: http://127.0.0.1:4450/wd/hub
    - browser: firefox
      selenium_hub_url: http://127.0.0.1:4451/wd/hub
  screenshots:
    dir: env:SCREENS_DIR
    before: none  # none | all | fail
    after: fail  # none | all | fail

tests:
  - get_ok:
      url: /
      wait:
        - has: css:form
  - has:
      css: not-existing
      timeout: 0.2
  - wait:
    - has: css:body
    - has: css:form
