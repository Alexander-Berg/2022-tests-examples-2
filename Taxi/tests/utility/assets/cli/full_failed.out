1..2  # full_failed.yml
  1..3  # full_failed chrome 800x600
  ok 1 - Get "/"
  not ok 2 - Has not-existing
  # Page has no element with locator "not-existing"
  # Waiting failed with <N> attempts
    1..2
    ok 1 - Has body
    ok 2 - Has form
  ok 3 - Wait while subtests will be resolved
  # Looks like you failed 1 tests of 3
not ok 1 - full_failed chrome 800x600
  1..3  # full_failed firefox 800x600
  ok 1 - Get "/"
  not ok 2 - Has not-existing
  # Page has no element with locator "not-existing"
  # Waiting failed with <N> attempts
    1..2
    ok 1 - Has body
    ok 2 - Has form
  ok 3 - Wait while subtests will be resolved
  # Looks like you failed 1 tests of 3
not ok 2 - full_failed firefox 800x600
# Looks like you failed 2 tests of 2
